options.go

```diff
@@ -56,6 +56,7 @@ type Options struct {
 	GitHubTeam               string   `flag:"github-team" cfg:"github_team" env:"OAUTH2_PROXY_GITHUB_TEAM"`
 	GitHubRepo               string   `flag:"github-repo" cfg:"github_repo" env:"OAUTH2_PROXY_GITHUB_REPO"`
 	GitHubToken              string   `flag:"github-token" cfg:"github_token" env:"OAUTH2_PROXY_GITHUB_TOKEN"`
+	GitHubUsers              []string `flag:"github-user" cfg:"github_user" env:"OAUTH2_PROXY_GITHUB_USER"`
```

Comment:
To be consistent with other list options, will also need to update the docs
```suggestion
	GitHubUsers              []string `flag:"github-user" cfg:"github_users" env:"OAUTH2_PROXY_GITHUB_USERS"`
```

Reply:
https://github.com/oauth2-proxy/oauth2-proxy/pull/421/commits/8952cfb01c99661f478465790477db58e29720b1
and updated docs: https://github.com/oauth2-proxy/oauth2-proxy/pull/421/commits/be7f969902b11d75b0af877a153815d4fc912fdf.